src/plugins/intel_gpu/src/kernel_selector/cl_kernels/permute_f_y_axes.cl

```diff
@@ -95,7 +95,7 @@ KERNEL (permute_f_y_axes)(
         __attribute__((opencl_unroll_hint(TILE_SIZE)))
         for (int i = 0; i < TILE_SIZE; ++i) {
             const int x_idx = x_begin + i;
-            const int f = f_begin + j;
+            const int f = (f_begin + j) % INPUT0_FEATURE_NUM;
```

Comment:
Could you please describe the situation, where we need to use this remainder? (some specific shapes?)

Reply:
16.32.128.512 b_fs_yx_fsv32
32.32.16.4 b_fs_yx_fsv16